### ISSUE TITLE
srcset causes infinite loop

### ISSUE THREAD

**Issue by joppuyo** [niutech/jxl.js#5]
Having jxl files in srcset attribute causes infinite loop.

Repo for repro:
https://github.com/joppuyo/jxl-js-responsive

PNG example (this works like it should)
https://joppuyo.github.io/jxl-js-responsive/srcset-png.html

JXL example (infinite loop, image doesn't display)
https://joppuyo.github.io/jxl-js-responsive/srcset-jxl.html

I think if srcset attribute exists, the blob uri should be replaced in the srcset attribute rather than src attribute.

**Comment by niutech** [comment]
Good catch, thanks! I've fixed it.

**Comment by joppuyo** [comment]
Thank you for the fast response!

I tested the latest version and srcset is now working! The only issue I noticed that while it works correctly when the image is loaded (desktop browser gets a large image and mobile device gets a smaller image), if you resize the browser window and make it bigger, the jxl.js image doesn't update to larger version of the image while native srcset does.

It's not a big issue but slight deviation from the native behavior.
